test/e2e/storage/persistent_volumes-local.go

```diff
@@ -349,7 +349,7 @@ var _ = utils.SIGDescribe("PersistentVolumes-local", func() {
 			framework.ExpectNoError(err)
 
 			err = e2epod.WaitTimeoutForPodRunningInNamespace(ctx, config.client, pod.Name, pod.Namespace, f.Timeouts.PodStart)
-			framework.ExpectError(err)
+			gomega.Expect(err).To(gomega.MatchError(framework.ErrFailure))
```

Comment:
`e2epod.WaitTimeoutForPodRunningInNamespace` + `expect error` -> this should be a `gomega.Consistently`.

Checking for `framework.ErrFailure` is too unspecific.

Reply:
got it. will refine later. thanks your reply.